scripts/ethcommands.ts

```diff
@@ -411,8 +411,8 @@ export const createERC20Command = {
           .toString()
       );
 
-      const l1GatewayRouter = new ethers.Contract(l1l2tokenbridge.l2Network.tokenBridge.l1GatewayRouter, L1GatewayRouter.abi, deployerWallet);
-      await (await token.functions.approve(l1l2tokenbridge.l2Network.tokenBridge.l1ERC20Gateway, ethers.constants.MaxUint256)).wait();
+      const l1GatewayRouter = new ethers.Contract(l1l2tokenbridge.l2Network.tokenBridge.parentGatewayRouter, L1GatewayRouter.abi, deployerWallet);
+      await (await token.functions.approve(l1l2tokenbridge.l2Network.tokenBridge.parentErc20Gateway, ethers.constants.MaxUint256)).wait();
```

Comment:
(it's not that I know what I'm talking about, but) I tried searching for `l1GatewayRouter` and `parentGatewayRouter` keys in our `nitro` repository; the former occurs only in contract tests (like https://github.com/OffchainLabs/nitro-contracts/blob/main/test/e2e/orbitChain.ts#L64), while the latter doesn't appear at all

my question is: are these new keys somehow autogenerated? also, should we update the contract tests?

I don't know the context, so just asking to understand the change a tad better 😅